code/game/machinery/telecomms/ntsl2.dm

```diff
@@ -356,14 +357,14 @@ GLOBAL_DATUM_INIT(nttc_config, /datum/nttc_configuration, new())
 					S.speaking = GLOB.all_languages[setting_language]
 
 	// Regex replacements
-	if(islist(regex) && regex.len > 0)
-		for(var/datum/multilingual_say_piece/S in message_pieces)
-			var/new_message = S.message
-			for(var/reg in regex)
-				var/replacePattern = pencode_to_html(regex[reg])
-				var/regex/start = regex("[reg]", "gi")
-				new_message = start.Replace(new_message, replacePattern)
-			S.message = new_message
+	// if(islist(regex) && regex.len > 0)
```

Comment:
Not the time but surely this should be a multiline comment

Reply:
I think these are a little easier because you can just CTRL + / in VSCode to add/remove them. But that may perhaps be cleaner.